include/wx/qt/choice.h

```diff
@@ -72,6 +72,7 @@ class WXDLLIMPEXP_CORE wxChoice : public wxChoiceBase
     virtual void DoDeleteOneItem(unsigned int pos);
 
     QComboBox *m_qtComboBox;
+    long m_style;
```

Comment:
This is unnecessary, the style is already stored in `wxWindowBase::m_windowStyle`, so it would just duplicate it.